### DISCUSSION TITLE
Trying to use `mod` to load a local file as a module

### DISCUSSION THREAD

**Discussion by frankitox** [evcxr/evcxr#375] [Q&A]
Hi! I was just trying to run a simple rust project with the REPL but I found a problem. First I run `evcxr`(0.17.0) at the root dir of my project, then in the REPL do `mod constants;` and get the error `[E0583] Error: file not found for module constants.` But I do have a file in `src/constants.rs`. Is there any way to tell `evcxr` about this file?

**Comment by frankitox** [comment]
Ok, I just found evcxr#70. Also from the REPL I checked for `:last_compile_dir` which returns a tmp folder in which I could symlink the file and it will work, but it's just a pretty shitty workaround.

**Comment by davidlattimore** [comment]
If you'd like a slightly less hacky workaround, you can also do something like this:

```rust
#[path = "/home/user/tmp/foo.rs"] mod foo;
```
